fah_xchem/analysis/diffnet.py

```diff
@@ -0,0 +1,343 @@
+from collections import defaultdict
+import logging
+from typing import Dict, List, Optional, Tuple
+
+import networkx as nx
+import numpy as np
+from scipy.special import logsumexp
+
+from ..schema import (
+    Compound,
+    CompoundAnalysis,
+    CompoundMicrostate,
+    Microstate,
+    MicrostateAnalysis,
+    PointEstimate,
+    TransformationAnalysis,
+)
+from .exceptions import AnalysisError, ConfigurationError, InsufficientDataError
+
+
+def pIC50_to_DG(pIC50: float, s_conc: float = 375e-9, Km: float = 40e-6) -> float:
+    """
+    Converts IC50 (in M units) to DG
+    Parameters
+    ----------
+    pIC50 : float
+        pIC50
+    s_conc : float, default=375E-9
+        Substrate concentration in M
+    Km : float, default=40E-6
+        Substrate concentration for half-maximal enzyme activity
+    Returns
+    -------
+    PointEstimate
+        Dimensionless free energy (in kT)
+    """
+    ic50 = 10 ** -pIC50
+
+    if ic50 > 1e-5:
```

Comment:
We should warn only if `ic50 > 200e-6` (the dynamic range of the assay)

Reply:
Updated in 724ac1d.